### PR TITLE
Update free-courses-en.md

### PR DESCRIPTION
**Pull request by yashsvi-30**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-en.md [hunk 1]
```diff
@@ -114,6 +115,11 @@
 * [Shell Scripting Tutorial](https://www.youtube.com/playlist?list=PL7B7FA4E693D8E790)
 
 
+### Bootstrap
+
+* [Rapid website design with Bootstrap](https://www.udemy.com/course/responsive-website-design) - Laurence Svekis (Udemy)
```
Comment (SethiPandi) [inline free-courses-en.md:120]: Could you please write the title of the resource in title case, like in the actual webpage?

**Review by SethiPandi — CHANGES_REQUESTED** [review]
Looks nice, I've just left a small comment to keep things tidy.

File: free-courses-en.md (hunk 1 above)
Comment (eshellman) [reply free-courses-en.md:120]: It's common for library catalogs to capitalize differently. Proper nouns, though, like JavaScript should follow the resource - I know, it's obscure, but we need to pick our battles.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**